src/test/kotlin/it/pagopa/wallet/controllers/WalletControllerTest.kt

```diff
@@ -1333,6 +1338,172 @@ class WalletControllerTest {
             .expectBody()
     }
 
+    @Test
+    fun testCreateWalletOkWhenLoggingEventFails() = runTest {
+        /* preconditions */
+        val pairLoggedActionUri =
+            Pair(
+                LoggedAction(WALLET_DOMAIN, WalletAddedEvent(WALLET_DOMAIN.id.value.toString())),
+                webviewPaymentUrl
+            )
+        given { walletService.createWallet(any(), any(), any(), any()) }
+            .willReturn(mono { pairLoggedActionUri })
+        given { walletEventSinksService.tryEmitEvent(any<LoggedAction<Wallet>>()) }
+            .willThrow(RuntimeException())
+        given { loggingEventRepository.saveAll(any<Iterable<LoggingEvent>>()) }
+            .willReturn(Flux.empty())
+        /* test */
+        webClient
+            .post()
+            .uri("/wallets")
+            .contentType(MediaType.APPLICATION_JSON)
+            .header("x-user-id", UUID.randomUUID().toString())
+            .header("x-client-id", "IO")
+            .bodyValue(WalletTestUtils.CREATE_WALLET_REQUEST)
+            .exchange()
+            .expectStatus()
+            .isCreated
+            .expectBody()
+            .jsonPath("$.walletId")
+            .value<String> { walletId ->
+                // Assert that the walletId is as expected
+                assert(walletId.startsWith(WALLET_DOMAIN.id.value.toString().trim())) {
```

Comment:
```suggestion
                assertTrue(walletId.startsWith(WALLET_DOMAIN.id.value.toString().trim())) {
```

use junit jupiter assertTrue method here

Reply:
Done in [a18f90a](https://github.com/pagopa/pagopa-wallet-service/pull/195/commits/a18f90a83a44c00163fc07e09c48a4c586e00661)